common/src/com/elikill58/negativity/common/protocols/Scaffold.java

```diff
@@ -86,4 +88,29 @@ public void onPacket(PacketReceiveEvent e) {
 			});
 		}
 	}
+	
+	
+	public static WeakHashMap<NegativityPlayer, Boolean> wasLastPacketArmAnimation = new WeakHashMap<>();
+	@EventListener
+	public void onPacketEvent(PacketEvent event) {
+		NegativityPlayer data =NegativityPlayer.getNegativityPlayer(event.getPlayer());
+		if(event.getPacket().getPacketType() == PacketType.Client.BLOCK_PLACE) {
+			if(wasLastPacketArmAnimation.get(data) == false) {
+				Negativity.alertMod(ReportType.VIOLATION,data.getPlayer(), this, 56, "NoSwing", "Last packet was not an ArmAnimation packet");
```

Comment:
Check name have to be like "no-swing" to correspond to others.
56% of reliability is below the default needed reliability (60), so never showed.
No check if check is active